### PR TITLE
Adding a new scenario for playtest on September 20th

### PR DESCRIPTION
**Pull request by TheGarron**
Implemented the prototype for classroom support on teacher moments. the new link displays on the demo list as "dysfunctional helping" as seen in the attached image.
screen shot 2018-09-19 at 2 28 55 pm

### CODE REVIEW

File: client/src/app.jsx [hunk 1]
```diff
@@ -63,6 +63,7 @@ export default createReactClass({
     '/teachermoments/sub': 'messagePopupPairs',
     '/teachermoments/jayden': 'jaydenScenario',
     '/teachermoments/rosa': 'rosaScenario',
+    '/teachermoments/dysfunc': 'dysfuncScenario',
```
Comment (TheGarron) [inline client/src/app.jsx]: this maps the URL to the scenario in the teacher moments application

File: client/src/home/demos_page.jsx [hunk 2]
```diff
@@ -35,10 +35,11 @@ export default class extends React.Component {
           {this.renderScenarioItem(<a href="/equity/climate?fromdemos">Gendered or racialized student comments</a>)}
           {this.renderScenarioItem(<a href="/teachermoments/smith?fromdemos">Noticing student belonging in the classroom</a>)}
           {this.renderScenarioItem(<a href="/teachermoments/sub?fromdemos">Positioning students in the classroom</a>)}
-          {this.renderScenarioItem(<a href="/teachermoments/rosa?fromdemos">Talking about identity</a>)}
+          {this.renderScenarioItem(<a href="/teachermoments/rosa?fromdemos">Talking about identity</a>)}          
           {this.renderScenarioItem(<a href="https://swipe-right-for-cs.herokuapp.com/play?fromdemos">Connecting student strengths to academics</a>)}
           {this.renderScenarioItem(<a href="/teachermoments/jayden?fromdemos">Encouraging student growth</a>)}
           {this.renderScenarioItem(<a href="/teachermoments/csfair?fromdemos">Computer science projects</a>)}
+          {this.renderScenarioItem(<a href="/teachermoments/dysfunc?fromdemos">Dysfunctional Helping</a>)}
```
Comment (TheGarron) [inline client/src/home/demos_page.jsx]: this change adds a link to the demo page in teacher moments to the new scenario "dysfunctional helping"

Comment (kevinrobinson) [reply client/src/home/demos_page.jsx]: sweet!

**Comment by TheGarron** [comment]
selfie

**Pull request merged**